modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/portlet/action/EditUserMVCActionCommand.java

```diff
@@ -608,14 +609,38 @@ private long _getListTypeId(
 		String parameterValue = ParamUtil.getString(
 			portletRequest, parameterName);
 
-		if (Validator.isNull(parameterValue)) {
-			return 0;
+		if (Validator.isNotNull(parameterValue)) {
+			ListType listType = _listTypeLocalService.addListType(
+				companyId, parameterValue, type);
+
+			return listType.getListTypeId();
+		}
+
+		User currentUser = _portal.getUser(portletRequest);
+		User selUser = _portal.getSelectedUser(portletRequest);
```

Comment:
sorry @wanderlast , can we rename also this to `User selectedUser` to be aligned with the method name (`getSelectedUser`)